build/get-operator-sdk.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+BASE_URL="https://github.com/operator-framework/operator-sdk/releases/download"
+
+# check if version is specified
+if [ "$#" -ne 1 ]
+then
+   echo "version is required"
+   exit 1
+fi
+VERSION="$1"
+
+# cd into the build directory
+cd "$(dirname "${0}")" || exit
+
+# Check if binary exists and is of correct version
+if [ -f ./operator-sdk ] && ./operator-sdk version | grep "$VERSION"
+then
+	exit 0
+fi
+
+# Choose the version suitable for platform
+if [[ "$OSTYPE" == "darwin"* ]]
+then
+	URL="$BASE_URL/$VERSION/operator-sdk-$VERSION-$(uname -m)-apple-darwin"
+elif [[ "$OSTYPE" == "linux-gnu"* ]]
```

Comment:
Let's default to the linux binary URL and overwrite it if darwin is detected. Right now, this logic could lead to an empty `$URL` variable depending on what `$OSTYPE` returns

Reply:
That is correct. I'll make the changes. Thank you :)